pkgs/development/interpreters/clisp/default.nix

```diff
@@ -112,22 +106,6 @@ stdenv.mkDerivation {
     find . -type f | xargs sed -e 's/-lICE/-lXau &/' -i
   '';
 
-  preConfigure = lib.optionalString stdenv.hostPlatform.isDarwin (
-    ''
-      (
-        cd src
-        autoreconf -f -i -I m4 -I glm4
-      )
-    ''
-    + lib.concatMapStrings (x: ''
-      (
-        root="$PWD"
-        cd modules/${x}
-        autoreconf -f -i -I "$root/src" -I "$root/src/m4" -I "$root/src/glm4"
-      )
-    '') (builtins.filter shouldReconfModule withModules)
-  );
```

Comment:
was this 2.50 only?

Reply:
Well, it was Darwin-specific, so if it builds on Darwin without that, should be fine to drop?